curriculum/challenges/english/01-responsive-web-design/applied-accessibility/know-when-alt-text-should-be-left-blank.english.md

```diff
@@ -7,7 +7,7 @@ videoUrl: 'https://scrimba.com/c/cM9P4t2'
 
 ## Description
 <section id='description'>
-In the last challenge, you learned that including an <code>alt</code> attribute on img tags is mandatory. However, sometimes images are grouped with a caption already describing them, or are used for decoration only. In these cases <code>alt</code> text may seem redundant or unnecessary.
+In the last challenge, you learned that including an <code>alt</code> attribute on <code>img</code> tags is mandatory. However, sometimes images are grouped with a caption already describing them, or are used for decoration only. In these cases <code>alt</code> text may seem redundant or unnecessary.
```

Comment:
@moT01, @ojeytonwilliams , @thecodingaviator While we are making changes to this particular section, should it be "on <code>img </code> tags is mandatory" or "in <code>img </code> tags is mandatory"?

Reply:
I think `on img tags` works - `in` sounds like it would be in between the opening and closing tags, even though `img `doesn't have a closing tag. Could use `with`... `including an alt attribute with img tags is mandatory`

Reply:
How about `including an <code>alt</code> attribute when using <code>img</code> tags is mandatory.`?

Reply:
I think @thecodingaviator's version works well.  By getting to this challenge they've just demonstrated they can put the attribute in the right place, so I don't think explicit guidance is necessary.

Reply:
I am fine with @thecodingaviator's version also.

Reply:
I also like @thecodingaviator's version.

---
@adamthahir Would you mind helping us update the PR with the change @thecodingaviator suggested?

Reply:
Alright, I have updated it as per the suggestion.

--
P.S: Sorry for not being more involved. I'm just getting started.